web/client/components/geostory/layouts/sections/Immersive.jsx

```diff
@@ -50,6 +50,9 @@ const Immersive = ({
     const hideContent = focusedContent && focusedContent.hideContent && (get(focusedContent, "target.id") === contentId);
     const visibility = hideContent ? 'hidden' : 'visible';
     const expandableBackgroundClassName = expandableMedia && background && background.type === 'map' ? ' ms-expandable-background' : '';
+    const overlayStoryTheme = storyTheme?.overlay || {};
+    const DEFAULT = 'default';
```

Comment:
With the new change on the theme selector now storyTheme contains `general` and `overlay` objects. The `general` is still needed by the background component. So we should pass `storyTheme?.overlay` to the SectionContents while `storyTheme?.general` to Background component. Please verify that both picker in the settings are working for background and overlay text.